src/content/resources/breaking-changes.md

```diff
@@ -68,6 +76,59 @@ don't include the section header.
 #### (Dart VM, Pub, Linter, `dart2js`, etc)
 {% endcomment %}
 
+## 3.9.0
+
+### Language {:.no_toc}
+
+- {{versioned}}
+  Null safety is now assumed when computing
+  type promotion, reachability, and definite assignment.
+  As a result, improved dead code analysis might
+  cause new analyzer diagnostics to trigger on
+  existing code that previously passed analysis.
+
+## 3.8.0
+
+### Libraries {:.no_toc}
+
+#### `dart:html`
+
+- {{removed}}
+  The `Element.created` constructor has been removed.
+- Native classes in `dart:html`, such as `HtmlElement`,
+  [can no longer be extended][53264].
+
+[53264]: {{site.repo.dart.sdk}}/issues/53264
+
+### Tools {:.no_toc}
+
+#### Analyzer
+
+- {{deprecated}}
+  The [`avoid_null_checks_in_equality_operators`][] lint rule
+  is deprecated and should be removed from `analysis_options.yaml` files.
+
+[`avoid_null_checks_in_equality_operators`]: /tools/linter-rules/avoid_null_checks_in_equality_operators
+
+#### Development JavaScript compiler (DDC)
+
+- Now throws a runtime error when a redirecting factory constructor is
+  torn off and one of its optional non-nullable parameters is provided no value.
```

Comment:
Maybe `removed` instead of `torn off`.

Reply:
This "torn off" is referring to a Dart functionality called [function tear offs](https://dart.dev/language/functions#tear-offs). I updated this terminology to cross link to the respective docs in case it's helpful for other developers.

Thanks for the review!

Reply:
Oh neat, I had no idea this was a feature. Thanks for that link.